include/turtle/detail/mutex.hpp

```diff
@@ -110,25 +108,25 @@ namespace detail
     {
         static Result abort()
         {
-            scoped_lock _( error_mutex );
+            scoped_lock _( error_mutex::inst() );
```

Comment:
```
In file included from test_sequence.cpp:10:
In file included from ../include/turtle/sequence.hpp:13:
In file included from ../include/turtle/detail/sequence_impl.hpp:13:
../include/turtle/detail/mutex.hpp:111:28: error: use of undeclared identifier 'error_mutex'; did you mean 'error_mutex_t'?
            scoped_lock _( error_mutex::inst() );
                           ^~~~~~~~~~~
```
and similar errors reported clang https://travis-ci.org/mat007/turtle/jobs/455635617

Reply:
Ouch, you are right. Should have been `error_mutex_t::inst()`